### ISSUE TITLE
Complete localization Gantry + Template

### ISSUE THREAD

**Issue by bartcz** [gantry/gantry5#2302]
Hello,

I want to completely translate Gantry Framework and all Particles and Atoms and asic template Helium into the Czech language.

Please advise wherever it is possible in which files to translate. I recently examined the Gantry Framework and the basic Helium template and started translating it. Translation phrases are not only in INI files (as far as I know).

It is not possible to change the language in most of the setting in the Gantry 5 framework as well as the field names and tooltip text in the particles. They are hardcoded in English. This is the only way it can be done (for now).
I do understand that it is not perfect, but this is a limitation that comes from the Gantry 5 framework, not from our templates and particles.

Thank you for your help.

Regards,

Bart

**Comment by dcsjapan** [comment]
I agree that the hardcoded language is a problem. I'm trying to create a Joomla site in a language other than English, and the theme I purchased from RocketThemes won't allow me to set particles to use a date format other than English. The month name gets translated by a language override somewhere, but the result looks nothing like a natural date in the frontend language. As a result I'm forced to avoid displaying dates in particles.

Gantry really needs to be internationalized so that both backend and frontend can be translated by users. But at the very least, it should respect Joomla's internationalization of things like date/time display.
